docker/openemr/5.0.2/php.ini

```diff
@@ -817,7 +849,7 @@ max_file_uploads = 20
 
 ; Whether to allow the treatment of URLs (like http:// or ftp://) as files.
 ; http://php.net/allow-url-fopen
-allow_url_fopen = On
+allow_url_fopen = Off
```

Comment:
@danehrlich1 ,
This looks suspicious since fopen is used frequently in openemr.